hoover/search/management/commands/doctor.py

```diff
@@ -0,0 +1,117 @@
+        return None
+
+
+class Command(BaseCommand):
+    help = "Sanity check for search. Run with no arguments."
+
+    def handle(self, *args, **options):
+        checkers = [
+            ('python', self.check_python, False),
+            ('database', self.check_database, False),
+            ('elasticsearch', self.check_es, False),
+            ('events_dir', self.check_events_dir, settings.HOOVER_EVENTS_DIR is None)
+        ]
+
+        have_errors = False
+        for name, check_fun, skip in checkers:
+            if skip:
+                self.print_message("Skipping the check for " + name + ".")
+            else:
+                self.print_message("Checking " + name + ".")
+                result = check_fun()
+                if result:
+                    self.print_success(' ' * 9 + name + " ok.")
+                else:
+                    have_errors = True
+                    self.print_error(name + " failed the check.")
+            self.print_message('')
+
+        if have_errors:
+            self.print_error("The setup has failed some checks.")
+            self.print_error("For more information please see")
+            self.print_error("https://github.com/hoover/search/blob/master/Readme.md")
+            sys.exit(1)
+        else:
+            self.print_success("All checks have passed.")
+
+    def check_python(self):
+        if sys.version_info[0] != 3 or sys.version_info[1] < 5:
+            self.print_error("The Python version supplied is {}.".format(sys.version))
+            self.print_error("Hoover needs at least Python 3.5 to work.")
+            self.print_error("Please use a supported version of Python.")
+            return False
+        return True
+
+    def check_database(self):
+        db_conn = connections['default']
+        try:
+            c = db_conn.cursor()
+        except OperationalError:
+            self.print_error("The database settings are not valid.")
+            self.print_error("Please check the database access data under DATABASES.")
+            return False
+        return True
+
+    def check_es(self):
+        es_link = settings.HOOVER_ELASTICSEARCH_URL
+        content = http_get_content(es_link)
+
+        if not content:
+            self.print_error("Could not connect to elasticsearch using")
+            self.print_error("the link supplied in HOOVER_ELASTICSEARCH_URL.")
+            return False
+
+        try:
+            data = json.loads(content)
+        except json.JSONDecodeError:
+            self.print_error("elasticsearch's response could not be decoded.")
+            self.print_error("Please restart the elasticsearch server and try again.")
+            return False
+
+        version_string = data['version']['number']
+        version_string = re.sub(r'[^\d\.]+', '', version_string)
+        version = tuple(int(x) for x in version_string.split('.'))
+
+        if not ELASTICSEARCH_MIN_VERSION <= version <= ELASTICSEARCH_MAX_VERSION:
```

Comment:
Actually we should allow any elasticsearch version 2.*, in case they release 2.5, which should not break any API.